.travis-setup.sh

```diff
@@ -35,42 +35,46 @@ sudo apt-get -qq install valgrind lcov uuid-dev pkg-config \
   zlib1g-dev libffi-dev gettext libpcre3-dev cppcheck \
   libmnl-dev
 
+function compile {
```

Comment:
I did something very similar here:

- https://github.com/jodh-intel/cc-oci-runtime/blob/run-functional-tests-in-travis/.travis-setup.sh#L42